Slides: Software Defined HPC with Open OnDemand

jprorama opened the pull request:
Slides presented at GOOD 2025 conference on March 19, 2024 describing the use of Open OnDemand as part of a software defined infrastructure to access campus HPC resources.

Thank you for submitting to the GOOD Conference Proceedings!

Please name your PR with `<type>: <title>`, where `<type>` is either `Paper`, `Slides`, or `Poster` and `<title>` is the title of your submission. An editor will then add the `draft` label to your PR and GitHub Actions will run automated checks and create a preview build.

See the https://good-proceedings.curve.space/author-instructions for full submission instructions.

jprorama committed:
Software Defined HPC with Open OnDemand

Slides presented at GOOD 2025 conference on March 19, 2024 describing
the use of Open OnDemand as part of a software defined infrastructure
to access campus HPC resources.

davidecarlson commented:
@rowanc1 Do we need to worry about the failed check, or can this slides submission be approved?

Thanks!

rowanc1 commented:
The PR is failing because there isn't any metadata. Adding a `myst.yml` would clean things up if it has the title, authors, etc.

Up to you on if you want to capture/create this after and just merge.

fwkoch committed:
✨ Add jpr-software-defined-hpc myst.yml

fwkoch committed:
🔧 Remove conference description from abstract

fwkoch commented on slides/jpr-software-defined-hpc/myst.yml:
```diff
@@ -0,0 +1,20 @@
+      name: The University of Alabama at Birmingham
+      department: Research Computing Development Team
+  abstract: |
+    We describe the use of Open OnDemand as part of a software defined
+    infrastructure to access campus HPC resources.
```
I pulled this from the commit message. The authors may want to provide a more complete abstract.

fwkoch reviewed (commented):
I've added the `myst.yml` metadata file, based on contents of the slides. Wouldn't hurt if the authors took a look but not absolutely necessary.

Regarding the failing action, the github actions are only intended for papers. I've created a different `slides` label we can use for these PRs (instead of `approved` which triggers paper-related checks and builds).

jprorama committed:
Update slides and myst.yml

Slides had wrong location for event.
Updated myst.yml to improve abstract and provide
Non-abbreviated affiliation.
